support-frontend/assets/pages/digital-subscription-checkout/components/thankYou/appsSection.jsx

```diff
@@ -122,7 +134,11 @@ const AppsSection = ({ countryGroupId }: PropTypes) => (
             nudgeIcon
             aria-label="Click to download the app on the Google Play store"
             href={androidAppUrl}
-            onClick={sendTrackingEventsOnClick('checkout_thankyou_play_store', 'DigitalPack', null)}
+            onClick={sendTrackingEventsOnClick({
```

Comment:
Same question

Reply:
Same as above, I'll update to include this. 👍